04_009/server/src/app.ts

```diff
@@ -0,0 +1,38 @@
+import express, { Application } from 'express';
+import errorMiddleware from '@/middlewares/error-middleware';
+import loaders from './loaders';
+import router from './routes';
+import CustomError from './error/custom-error';
+import { NOT_FOUND_ERROR } from './constants/error';
+
+class App {
+  app: Application;
```

Comment:
```suggestion
  private app: Application;
```
타입스크립트를 사용하기 때문에 이렇게 private으로 접근을 제한해주면 더 좋을 것 같아요!